src/main/java/org/mockito/internal/configuration/plugins/PluginLoader.java

```diff
@@ -51,20 +59,22 @@ public PluginLoader(PluginSwitch pluginSwitch) {
         try {
             resources = loader.getResources("mockito-extensions/" + service.getName());
         } catch (IOException e) {
-            throw new MockitoException("Failed to load " + service, e);
+            // use RuntimeException instead MockitoException, because MockitoException depends on plugins and can
+            // break stack track
+            throw new RuntimeException("Failed to load " + service, e);
         }
 
         try {
             String foundPluginClass = new PluginFinder(pluginSwitch).findPluginClass(Iterables.toIterable(resources));
             if (foundPluginClass != null) {
-                Class<?> pluginClass = loader.loadClass(foundPluginClass);
+                Class<?> pluginClass = Class.forName(foundPluginClass);
                 Object plugin = pluginClass.newInstance();
                 return service.cast(plugin);
             }
             return null;
         } catch (Exception e) {
             throw new MockitoConfigurationException(
-                    "Failed to load " + service + " implementation declared in " + resources, e);
+                                                           "Failed to load " + service + " implementation declared in " + resources, e);
```

Comment:
IDE is configured for another project formatting, in contribute guide I found only mention about spaces. So I've continued use my configuration